scripts/e2e.sh

```diff
@@ -22,8 +22,19 @@ export WIDGET_WEB_CLIENT_ID=0oa8ls36zUZj7oFJ2696
 
 export ORG_OIE_ENABLED=true
 
+# Run spec tests
 if ! yarn test:e2e; then
-  echo "e2e tests failed! Exiting..."
+  echo "e2e spec tests failed! Exiting..."
+  exit ${PUBLISH_TYPE_AND_RESULT_DIR_BUT_ALWAYS_FAIL}
+fi
+
+# Run feature tests
```

Comment:
We run feature tests in the same bacon e2e test. 
Additional env vars are required to run feature tests using the same yarn command.